### PR TITLE
Validation fixes for IE=Edge, & and CSS margins

### PR DESCRIPTION
**Pull request by ryanfeeley**

### CODE REVIEW

**Comment by jaredhirsch** [comment]
@ryanfeeley see shane's comments, mind updating the branch?

File: resources/email_templates/confirm.html.ejs [hunk 1]
```diff
@@ -100,7 +100,7 @@
                 <table border="0" cellpadding="0" cellspacing="0" width="600" style="margin-bottom: 60px;">
                   <tr>
                     <td align="left" valign="top" style="border-collapse: collapse; color: #d4d4d4; padding: 0">
-                      <%- format(gettext('<strong>Mozilla Persona.</strong> Simple sign-in from the non-profit behind Firefox. <a %s>Learn more&rarr;</a>'), ["href='https://login.persona.org/about' target='_blank' style='color:#ffa274'"]) %>
+                      <%- format(gettext('<strong>Mozilla Persona.</strong><br />Simple sign-in from the non-profit behind Firefox.<br /><a %s>Learn more&rarr;</a>'), ["href='https://login.persona.org/about' target='_blank' style='color:#ffa274'"]) %>
```
Comment (shane-tomlinson) [inline resources/email_templates/confirm.html.ejs:103]: We tried this before, I sent you a screen shot, and you said "ixnay". See https://github.com/mozilla/browserid/pull/3957#issuecomment-25904776

Comment (ryanfeeley) [inline resources/email_templates/confirm.html.ejs:103]: You’re right! Ignore.

On Oct 28, 2013, at 11:26 AM, Shane Tomlinson notifications@github.com wrote:

> In resources/email_templates/confirm.html.ejs:
> 
> > @@ -100,7 +100,7 @@
> >                  <table border="0" cellpadding="0" cellspacing="0" width="600" style="margin-bottom: 60px;">
> >                    <tr>
> >                      <td align="left" valign="top" style="border-collapse: collapse; color: #d4d4d4; padding: 0">
> > -                      <%- format(gettext('<strong>Mozilla Persona.</strong> Simple sign-in from the non-profit behind Firefox. <a %s>Learn more&rarr;</a>'), ["href='https://login.persona.org/about' target='_blank' style='color:#ffa274'"]) %>
> > -                      <%- format(gettext('<strong>Mozilla Persona.</strong><br />Simple sign-in from the non-profit behind Firefox.<br /><a %s>Learn more&rarr;</a>'), ["href='https://login.persona.org/about' target='_blank' style='color:#ffa274'"]) %>
> >   We tried this before, I sent you a screen shot, and you said "exnay". See #3957 (comment)
> 
> —
> Reply to this email directly or view it on GitHub.

Ryan Feeley
Product Designer, Identity
Mozilla UX
IRC: rfeeley

**Comment by seanmonstar** [comment]
@6a68 @ryanfeeley so whats the status here? do we merge? or close?

**Comment by callahad** [comment]
AFAICT, we should revert the `<br>` per Ryan's comment, and address the duplication from Shane's comment, then merge.

**Comment by callahad** [comment]
Actually, current dev looks like it has these changes. Closing.

**Pull request closed without merging**